types/ApiRequests.ts

```diff
@@ -308,6 +309,7 @@ export interface OrderPickupEvent {
   end: Date;
   description: string;
   orderLimit: number;
+  linkedEvent?: Event;
```

Comment:
I made this of type `Event`, should it instead be of type `UUID`?

Reply:
I think event is fine for the api response, saving us an extra api request